src/components/Routes/Routes.js

```diff
@@ -9,12 +11,18 @@ const Routes = () => (
         {sections.map(section => (
             <Route
                 key={section.key}
+                exact
                 path={section.path}
                 component={props => (
                     <section.component sectionKey={section.key} {...props} />
                 )}
             />
         ))}
+        <Route
+            path="/lock-exceptions/batch-deletion"
+            component={BatchDeleteLockExceptions}
+        />
+        <Route path="/lock-exceptions/add" component={AddLockException} />
```

Comment:
Why are these not in the `sections.conf.js`?

Reply:
As discussed in https://jira.dhis2.org/browse/DHIS2-11344, `sections.conf.js` is not a pattern typical of react apps (only the data admin and data quality apps use it). My thinking is that while removing it should be done in a separate PR, new components do not have to be included in it.

Reply:
:ok: Sounds good to me, could you add that as a comment to the code?

Reply:
:+1: